ci-operator/config/openshift/release/openshift-release-master__ocp-4.7-ci.yaml

```diff
@@ -0,0 +1,31 @@
+releases:
+  initial:
+    release:
+      channel: stable
+      version: "4.6"
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.7"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: metal-ipi-upgrade-e2e-46-stable-to-47-ci
```

Comment:
As discussed, I think we could drop the `ci` stream, since metal-ipi is currently applied only on the `nightly` one